packages/unlock-js/src/erc20.ts

```diff
@@ -93,6 +93,106 @@ export async function approveTransfer(
   return contract.approve(lockContractAddress, value)
 }
 
+interface TransferAuthorizationMessage {
+  from: string
+  to: string
+  value: any
+  validAfter: number
+  validBefore: number
+  nonce: string
+}
+
+export async function signTransferAuthorization(
+  erc20ContractAddress: string,
+  message: TransferAuthorizationMessage,
+  provider: ethers.providers.Provider,
+  signer: ethers.Signer
```

Comment:
Same here. Can we use an object so adding new arguments does not cause a breaking change>